_posts/2016-12-22-Rust-1.14.md

```diff
@@ -0,0 +1,356 @@
+[todomvc]: http://timryan.org/rust-todomvc/
+
+Speaking of platforms, a large number of platforms have gained additional
+support:
+
+For `rustc`:
+
+* `mips-unknown-linux-gnu`
+* `mipsel-unknown-linux-gnu`
+* `mips64-unknown-linux-gnuabi64`
+* `mips64el-unknown-linux-gnuabi64`
+* `powerpc-unknown-linux-gnu`
+* `powerpc64-unknown-linux-gnu`
+* `powerpc64le-unknown-linux-gnu`
+* `s390x-unknown-linux-gnu`
+
+And for `std`:
+
+* `arm-unknown-linux-musleabi`
+* `arm-unknown-linux-musleabihf`
+* `armv7-unknown-linux-musleabihf`
+
+If you're using one of these platforms, follow the instructions on the website
+to install, or add the targets to an existing installation with
+`rustup target add`.
+
+These platforms are all 'tier 2', please see our page on [platform support] for
+more details.
+
+[platform support]: https://forge.rust-lang.org/platform-support.html
+
+The landing of MIR over the last few releases means that a [number of
+improvements to compile times] have landed, with more coming in the future.
+Exact numbers are hard, as they depend on what code you're compiling, but
+broadly speaking, compile times should be the same or faster.
+
+[numer of improvements to compile times]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#compile-time-optimizations
+
+In the language, one small improvement has landed: support for [RFC 1492]. This small
+additional lets you use `..` in more places. Previously, say you had a struct like this:
+
+```rust
+struct Point {
+    x: i32,
+    y: i32,
+    z: i32,
+}
+```
+
+In any context where you're doing a pattern match, you could use `..` to ignore the
+parts you don't care about. For example:
+
+
+```rust
+let p = Point { x: 0, y: 1, z: 2 };
+
+match p {
+    Point { x, .. } => println!("x is {}", x),
+}
+```
+
+The `..` ignores `y` and `z`.
+
+Consider a similar `Point`, but as a tuple struct:
+
+```rust
+struct Point(i32, i32, i32);
+```
+
+Previously, you could use `..` to ignore all three elements:
+
+```rust
+let p = Point(0, 1, 2);
+
+match p {
+    Point(..) => println!("found a point"),
+}
+```
+
+But you could not use it to only ignore parts of the tuple:
+
+```rust
+let p = Point(0, 1, 2);
+
+match p {
+    Point(x, ..) => println!("x is {}", x),
+}
+```
+
+This was a inconsistency, and so with RFC 1492 stabilized, compiles fine as of
+this release. This applies to more situations than tuples; please see [the
+RFC][RFC 1492] for more details.
+
+[RFC 1492]: https://github.com/rust-lang/rfcs/blob/master/text/1492-dotdot-in-patterns.md
+
+#### Library stabilizations
+
+There have been a number of additions to the standard library, but they don't
+fit into particularly nice categories for this release. Here's the highlights:
+
+* [`println!()`, with no arguments, prints newline][36825].
+  Previously, an empty string was required to achieve the same.
+* [`Wrapping` impls standard binary and unary operators, as well as
+   the `Sum` and `Product` iterators][37356]
```

Comment:
actually the operator impls were already there, this just adds them for references